Yoast/wordpress-seo
# P2-53 Decrease the zip size

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Decreases the zip size.

## Relevant technical choices:

* In https://github.com/Yoast/wordpress-seo/commit/c33f8b80c95006f8cbadc53d6c239a4213d87c3f too many occurrences of `yoast-components` were changed to `configuration-wizard.scss`, also in places that didn't have to do with css (cc @JoseeWouters for context). In this PR, those changes are reverted. 

## Test instructions

This PR can be tested by following these steps:

* Build a zip with this code (you might need [this commit](https://github.com/Yoast/wordpress-seo/commit/27731c11248326b89c860d5231524a93eba57831) to be able to build it).
* Everything that uses draft-js and the replacement variable editor. This is the snippet editor with the replacement variables. Please test that in the metabox on posts/pages and terms.
* The translations should still function as before. Smoketest them.

## UI changes

## Documentation

## Quality assurance


Fixes #
